sphinx_gherkindoc/writer.py

```diff
@@ -368,7 +368,8 @@ def table(table: behave.model.Table, inline: bool = False) -> None:
     # but not having scenarios in it,
     # only exclude the feature (and its description, etc)
     # if include/exclude logic has been activated.
-    if not included_scenarios and (include_tags or exclude_tags):
+    # or in the event that there is an enmpty feature file.
+    if (not included_scenarios and (include_tags or exclude_tags)) or not included_scenarios:
```

Comment:
Reading this quickly, I think what this is say is:
`if (not A and (C or D)) or not A:`
Which should be equal to just `not A`??
I need to re-read this less quickly to see if that's right or not...

Reply:
I also didn't think throught that logic expression. `not A` should be enough . The `get_all_included_scenarios` should filter out based on the tags, so I'm not sure if that check was even necessary.